packages/components/label/src/label.tsx

```diff
@@ -80,14 +79,14 @@ const Label = (props: TLabelProps) => {
     <label
       css={css`
         > div {
-          font-weight: ${!props.isBold && designTokens.fontWeight500};
+          font-weight: ${props.fontWeight};
```

Comment:
I think this change should be different since now we can see a lot of changes in Percy related to `Label` component using a different `font-weight`.

If I'm not mistaken, the idea here is that, if the consumer didn't want to use `bold`, the `font-weight` to be used should be `500` instead of the default value `400`.
Also, now we're passing one of these values to this rule: `regular`, `medium` and `bold`, and only the latter is a supported value for that rule.

Also, I noticed we're removing the `isBold` property from this component, which would be a breaking change so we should avoid it.

I think we should try to keep the previous behaviour:
```
font-weight: ${(props.isBold !== true && props.fontWeight !== 'bold') && designTokens.fontWeight500}
```

@FilPob did you take a look at Percy? Does this make sense to you?

Reply:
@CarlosCortizasCT so in a previous comment you suggested to remove isBold from the `label.visualroute.jsx` file and I think I took that to mean it needed to be removed from the `label` component too. It looked like `isBold` there was being passed into the label component where it ultimately was used on a `Text.Detail` within that component. So you're saying `isBold` needs to be restored at the top level usage?

Reply:
> Also, I noticed we're removing the `isBold` property from this component, which would be a breaking change so we should avoid it.

@CarlosCortizasCT so in a previous comment you suggested to remove isBold from the label.visualroute.jsx file and I think I took that to mean it needed to be removed from the label component too. It looked like isBold there was being passed into the label component where it ultimately was used on a Text.Detail within that component. So you're saying isBold needs to be restored at the top level usage?

Reply:
I'm sorry for the misunderstanding.

We need to bear in mind the `Level` component is a public one; it can be consumed in isolation requiring [its own package](https://uikit.commercetools.com/?path=/story/components-label--label). So if we remove a property from it, all current consumers would be affected and that's why I think we should not do that.

It makes sense to me to not use the deprecated property (**isBold**) internally, but maybe it would make sense to keep one VRT where we use it exclusively to resemble current consumers expected behaviour.

Sorry for the back and forth. 😢 

We can have a chat about this anytime you want if you feel like we need more clarification.

Reply:
I am seeing two conflicting messages here.  

That we should keep `isBold` and not accept the `fontWeight` prop in the component api for `Label`:

> Also, I noticed we're removing the isBold property from this component, which would be a breaking change so we should avoid it.

And that we should accept `fontWeight`, but override the default to `500` if there isn't a value specified for the `fontWeight` prop on the `Label`.

> `font-weight: ${(props.isBold !== true && props.fontWeight !== 'bold') && designTokens.fontWeight500}`

So, here is my solution - which adds `isBold` back to the `Label` component api as a deprecated prop and sets the default `font-weight` to `500` if neither `isBold` or `fontWeight` is passed to `Label`

```js
font-weight: ${props.isBold !== true && props.fontWeight !== undefined && designTokens.fontWeight500};
```

@CarlosCortizasCT @FilPob let me know if this is the correct solution, or if you'd like to not deprecate `isBold` and remove the `fontWeight` prop from the `Label` api altogether.

Reply:
Agreed and updated here: 8797858fefbf364f1a510b3e96caeb4087fb145b

Reply:
Just leaving this here for posterity:

After talking with @CarlosCortizasCT, we have:

- deprecated the `isBold` prop, but not removed it from `Label`
- only allowed `medium` (`500`) or `bold` (`600`) as values for `fontWeight` for `Label`

Instead of implementing `font-weight: 500` by using a ternary in a css child selector that targets `Text.Detail`, I've refactored the component so that `fontWeight='medium'` is the default `fontWeight` prop for `label`, which will insure that the label's font weight is `500` unless the user specifies `bold`